checkov/helm/runner.py

```diff
@@ -126,16 +126,34 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
 
                 self.parse_helm_dependency_output(o)
 
-                try:
-                    # --dependency-update needed to pull in deps before templating.
-                    proc = subprocess.Popen([self.helm_command, 'template', '--dependency-update', chart_dir], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
-                    o, e = proc.communicate()
-                    logging.debug(
-                        f"Ran helm command to template chart output. Chart: {chart_meta['name']}. dir: {target_dir}. Output: {str(o, 'utf-8')}")
+                if runner_filter.var_files != None:
+                    var_files_helm_formatted = []
+                    for var in runner_filter.var_files:
+                        var_files_helm_formatted.append("--values")
+                        var_files_helm_formatted.append(var)
+                    try:
+                        # --dependency-update needed to pull in deps before templating.
+                        proc2 = subprocess.Popen([self.helm_command, 'template', '--dependency-update', chart_dir] + var_files_helm_formatted, stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
+                        o, e = proc2.communicate()
+                        logging.debug(
+                            f"Ran helm command to template chart output. Chart: {chart_meta['name']}. dir: {target_dir}. Output: {str(o, 'utf-8')}")
 
-                except Exception:
-                    logging.info(
+                    except Exception:
```

Comment:
@metahertz Missing an `as e:`?
```suggestion
                    except Exception as e:
```

Reply:
Wasnt accessing the error, but 100% should have included! Thanks.